repos.json

```diff
@@ -90,6 +90,9 @@
         "emmanuelrosa": {
             "url": "https://github.com/emmanuelrosa/nur-packages"
         },
+        "extends": {
+            "url": "https://github.com/ImExtends/nur-packages-template"
```

Comment:
Do you intend not to rename it to just `nur-packages`?

Reply:
I'm fine with it ! thanks for asking